en/docs/integrations/integration-concepts.md

```diff
@@ -29,15 +29,15 @@ A statement is a collection of code fragments that specifies the actions to be c
 
 Choreo supports the following statements:
 
- - **Log**: Logs an event with an information statement or an error that occurs in your Choreo integration.
- - **Variable**: Declares a variable and a value of a preferred data type (`string`, `int`, etc.) to use later in the logical process of the Choreo integration.
- - **Data Mapping**: Allows to create variables by visually mapping one or more variables. For more information, see [Data Mapping](../references/data-mapping.md).
- - **If**: Performs conditional execution.
- - **ForEach**: Includes a control flow statement in your Choreo integration to iterate over a list of items.
- - **While**: Continuously executes a block of statements in a loop based on a given `boolean` condition. 
- - **Respond**: Sends a specific response back to the client. For example, if the request was processed successfully, responding to the client with an `HTTP_OK`.
- - **Other**: Allows to write a single or a multiline code snippet in your Choreo integration.
+ - **Log:** Logs an event with an information statement or an error that occurs in your Choreo integration.
+ - **Variable:** Declares a variable and a value of a preferred data type (`string`, `int`, etc.) to use later in the logical process of the Choreo integration.
+ - **Data Mapping:** Allows you to create variables by visually mapping one or more variables. For more information, see [Data Mapping](../references/data-mapping.md).
+ - **If:** Performs conditional execution.
+ - **ForEach:** Includes a control flow statement in your Choreo integration to iterate over a list of items.
+ - **While:** Continuously executes a block of statements in a loop based on a given `boolean` condition. 
+ - **Respond:** Sends a specific response back to the client. For example, if the request was processed successfully, it responds to the client with an `HTTP_OK`.
+ - **Other:** Allows you to write a single or a multiline code snippet in your Choreo integration.
 
-## Ballerina expression syntax
+## Expression
```

Comment:
These are Ballerina expressions... and what we discuss here is the syntax.  We could maybe say, Expression syntax

Reply:
+1
Updated.